test/py2vs3.py

```
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
```

using the origina module should raise an exception at some point, or at least do a logging.error
and i think almost all these test can be simplified if not removed

---

trying to use py2 will already give an import error. I'll add a logging statement when you try to load the py3 version. I think I cleaned most of the code. We could also remove it, but things might break.

---

up to you

---

warning added